Gemfile

```diff
@@ -22,3 +22,5 @@ if RUBY_VERSION >= "3.1"
   gem "net-pop", require: false
   gem "net-smtp", require: false
 end
+
+gem "debug"
```

Comment:
Did you mean to add this?

Reply:
Yes! It's very handy to have installed when working on the project locally.